File: TwoSum_II.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        res = []
        l,r = 0,len(numbers)-1
        while l<r : 
            if numbers[l]+numbers[r]<target : 
                l+=1
            elif numbers[l]+numbers[r]>target : 
                r-=1
            else : 
                return [l+1,r+1]

File: TwoSum_II/TwoSum_II.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int l = 0;
        int r = numbers.length - 1; 
        while (l < r) {
            int sum = numbers[l] + numbers[r];
            if (sum < target) {
                l++;
            } else if (sum > target) {
                r--;
            } else {
                return new int[]{l + 1, r + 1};
            }
        }
        return new int[0];
    }
}